core/pkg/server/sender_test.go

```diff
@@ -46,7 +46,13 @@ func makeSender(client graphql.Client, recordChan chan *spb.Record, resultChan c
 	})
 	backend := server.NewBackend(logger, settings)
 	fileStream := server.NewFileStream(
-		backend, logger, observability.NewPrinter(), settings, nil)
+		backend,
+		logger,
+		nil,
+		observability.NewPrinter(),
+		settings,
+		nil,
```

Comment:
add comments about the argument for the nil values

Reply:
good point, done!

Reply:
oops, just saw the commit for this in my local branch... forgot to squash it into the PR

#8582